WordPressAuthenticator/Signin/LoginProloguePageViewController.swift

```diff
@@ -2,7 +2,11 @@ import UIKit
 import WordPressShared
 
 class LoginProloguePageViewController: UIPageViewController {
-    @objc var pages: [UIViewController] = []
+    // This property is a legacy of the previous UX iteration. It ought to be removed, but that's
+    // out of scope at the time of writing. It's now `private` to prevent using it within the
+    // library in the meantime
+    @objc private var pages: [UIViewController] = []
```

Comment:
```suggestion
    @available(*, deprecated, message: "This property is a legacy of the previous UX iteration and ought to be removed (but that was out of scope at the time of writing).")
    // Property is `private` to prevent using it within the library in the meantime, see deprecation message above
    @objc private var pages: [UIViewController] = []
```

Reply:
That's a nice idea, thanks.

I tried it locally, and got a lot (well, 10) warnings:

<img width="1127" alt="image" src="https://user-images.githubusercontent.com/1218433/157804403-aee0d879-1ae3-4ca8-98e5-c2e02edb4534.png">

_All the warnings in the mini map are because of the deprecated `page`._

Part of me welcomes them. "Yes! Make some noise in the build so we get rid of dead code ASAP." Another part of me doesn't want them because, while it's true that we ought to get rid of that code, there's no pressing necessity to do it and I wouldn't want other, legit warnings to get lost among them. 

Currently, I'm leaning toward not throwing the warnings, so I'm going to merge as is. Happy to revisit, though.